Docs/SpellCheckingConfiguration.md

```diff
@@ -25,22 +28,31 @@ You can provide these files in your own repository and by default they must be p
 
 When the spell checking process finishes with errors detected, the report commented automatically to the PR will give recommendations for external dictionaries that cover some of the unrecognized entries. These dictionaries can be added to your action/workflow configuration so they are included during spell-checking. Even if a dictionary covers a lot of unrecognized entries, it might not be suitable for your project and could leave you with a lot of false negative detections.
 
-- Before adding a dictionary, check if it's contextually relevant. For example, the Python and Typescript dictionaries seem to appear frequently in the recommendations for ASP.NET Core-based projects having mostly C# code, so be cautious if your project doesn't actually utilise that technology.
+- Before adding a dictionary, check if it's contextually relevant. For example, the Typescript dictionary seems to appear frequently in the recommendations for ASP.NET Core-based projects having mostly C# code, so be cautious if your project doesn't actually utilise that technology.
+- Only add one dictionary at a time so that recommendations are re-evaluated correctly. This makes it easier to keep your configuration at the necessary minimum, since some dictionaries are overlapping.
 - Also check if the contents of the dictionary make sense. Some recommended libraries seem to have a great number of entries that seem useless in light of how the spell checking process works now. That is likely due to those dictionary files being outdated and fundamental changes have been made to process since.
-- Try to avoid large dictionaries, like the one for C++: It has over 30 thousand entries, because it casts a net way too large and contains a ton of entries that should have been excluded by other means (e.g., a pattern, ignoring a line or a while file).
+- Try to avoid large dictionaries, like the one for C++: It has over 30 thousand entries, because it casts a net way too large and contains a ton of entries that should have been excluded by other means (e.g., a pattern, ignoring a line or a whole file).
```

Comment:
Explain why it's good to avoid large dictionaries.

Reply:
It's explained in this section from multiple angles, so please read the section as a whole, not just individual lines. What's missing for your understanding?

Reply:
I've read it, and I don't see it explaining how specifically _large_ (i.e. not overlapping, or ones causing false negatives; talking about the size, not content) dictionaries cause an issue. Is it a performance issue?

Reply:
OK, let's leave it like that.